### PR TITLE
Add release notes for v1.1.6 and v2.0-beta.20180312

### PR DESCRIPTION
**Pull request by jseldess**

### CODE REVIEW

**Comment by bdarnell** [comment]
Review status: 0 of 5 files reviewed at latest revision, 6 unresolved discussions, some commit checks failed.

---

*[_data/releases.yml, line 36 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70TEhaA-MAchX4Ivdd:-L70TEhaA-MAchX4Ivde:b-b0sifa) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/_data/releases.yml#L36)):*
> ```YAML
> - title: Testing Releases
>   releases:
>     - date: Mar 012, 2018
> ```

Remove the extra 0

---

*[releases/v1.1.6.md, line 35 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70TKVb773cgYbMvgJw:-L70TKVb773cgYbMvgJx:b-ooggde) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v1.1.6.md#L35)):*
> ```Markdown
> ### General Changes
> 
> - disabling diagnostics reporting also disables new version notification checks. [#23008][#23008]
> ```

Capitalize "Disabling".

---

*[releases/v1.1.6.md, line 62 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70TTzyBNOC36aYgv_0:-L70TTzyBNOC36aYgv_1:b-6nzv6e) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v1.1.6.md#L62)):*
> ```Markdown
>   format for arrays. [#22262][#22262]
> 
> ### Miscellaneous
> ```

Remove this?

---

*[releases/v1.1.6.md, line 67 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70Te5xSZDdSF4VcNcB:-L70Te5xSZDdSF4VcNcC:bpfzdw0) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v1.1.6.md#L67)):*
> ```Markdown
> #### Changes without release note annotation
> 
> - [#22663][#22663] [ac40ef74f][ac40ef74f] release-1.1: server: Don't send pgwire connections to the init GRPC server (Ben Darnell)
> ```

Bug fix: Prevent SQL connection errors while a node is starting up.

---

*[releases/v1.1.6.md, line 68 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70TXkktHL5s1SQXIJM:-L70TXkktHL5s1SQXIJN:b-6qwajk) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v1.1.6.md#L68)):*
> ```Markdown
> 
> - [#22663][#22663] [ac40ef74f][ac40ef74f] release-1.1: server: Don't send pgwire connections to the init GRPC server (Ben Darnell)
> - [#22826][#22826] [54fe87da8][54fe87da8] build: allow building with go1.10 (Xudong Zheng)
> ```

Don't mention this in the notes. 

---

*[releases/v2.0-beta.20180312.md, line 39 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70TvNyFzGR7OCvo05d:-L70TvNyFzGR7OCvo05e:b-dq2sv6) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v2.0-beta.20180312.md#L39)):*
> ```Markdown
> ### General Changes
> 
> - logging data is now flushed to files
> ```

More capitalization needed in this file.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2604)*

**Comment by bdarnell** [comment]
Review status: 0 of 5 files reviewed at latest revision, 7 unresolved discussions, some commit checks failed.

---

*[releases/v2.0-beta.20180312.md, line 84 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70USeqRVWdAx5L97rd:-L70USeqRVWdAx5L97re:b-we23pe) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v2.0-beta.20180312.md#L84)):*
> ```Markdown
>   debug page. [#23241][#23241]
> 
> ### Bug Fixes
> ```

cockroachdb/cockroach#23324 is included in this release, right? It needs a release note since it fixes the issue we warned about in the 0305 beta. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2604)*

**Comment by jseldess** [comment]
I haven't done any editing here, @bdarnell. Sorry for not being clear. Opening this PR early so @nstewart can take a look at what's in each release.

**Comment by jseldess** [comment]
*[releases/v1.1.6.md, line 67 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70Te5xSZDdSF4VcNcB:-L70b4LmwQJaynEyJLn9:bccuobg) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v1.1.6.md#L67)):*
@bdarnell, in our slack chat, you said:
> this one should copy the release note from 21682: Release note (CLI change): cockroach start --background now returns earlier for nodes awaiting the cockroach init command, facilitating use from automated scripts.

I'm not clear on whether to use one, the other, or both. Can you clarify?

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2604)*

**Comment by jseldess** [comment]
*[releases/v2.0-beta.20180312.md, line 84 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70USeqRVWdAx5L97rd:-L70bRY07cvrCRHzFGsy:b-8wx7zx) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v2.0-beta.20180312.md#L84)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary><blockquote>

cockroachdb/cockroach#23324 is included in this release, right? It needs a release note since it fixes the issue we warned about in the 0305 beta. 
</blockquote></details>

OK. It doesn't have a release note text, but I'll try to draft one.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2604)*

**Comment by bdarnell** [comment]
Review status: 0 of 5 files reviewed at latest revision, 7 unresolved discussions, some commit checks failed.

---

*[releases/v1.1.6.md, line 67 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70Te5xSZDdSF4VcNcB:-L70cnahRvw9Z8r7F94_:bt3wru7) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v1.1.6.md#L67)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

@bdarnell, in our slack chat, you said:
> this one should copy the release note from 21682: Release note (CLI change): cockroach start --background now returns earlier for nodes awaiting the cockroach init command, facilitating use from automated scripts.

I'm not clear on whether to use one, the other, or both. Can you clarify?
</blockquote></details>

Just this one (prevent sql connection errors while a node is starting up). I misread things earlier. 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2604)*

**Comment by bdarnell** [comment]
Review status: 0 of 5 files reviewed at latest revision, 7 unresolved discussions, some commit checks failed.

---

*[releases/v2.0-beta.20180312.md, line 84 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70USeqRVWdAx5L97rd:-L70hOYjdkWJEl7uwJ-h:baxsqrb) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v2.0-beta.20180312.md#L84)):*
<details><summary><i>Previously, jseldess wrote…</i></summary><blockquote>

OK. It doesn't have a release note text, but I'll try to draft one.
</blockquote></details>

I think we're also going to want cockroachdb/cockroach#23541 which is closely related, so the two can share one release note. cc @arjunravinarayan 

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2604)*

**Comment by jseldess** [comment]
Review status: 0 of 5 files reviewed at latest revision, 7 unresolved discussions.

---

*[_data/releases.yml, line 36 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70TEhaA-MAchX4Ivdd:-L70hsHA-UxZzGk-ttqv:boj1qgt) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/_data/releases.yml#L36)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary><blockquote>

Remove the extra 0
</blockquote></details>

Good catch. Done.

---

*[releases/v1.1.6.md, line 35 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70TKVb773cgYbMvgJw:-L70la74CFHNJUrjEUH3:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v1.1.6.md#L35)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary><blockquote>

Capitalize "Disabling".
</blockquote></details>

Done.

---

*[releases/v1.1.6.md, line 62 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70TTzyBNOC36aYgv_0:-L70hmwenFqs1XXsT9S-:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v1.1.6.md#L62)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary><blockquote>

Remove this?
</blockquote></details>

Done.

---

*[releases/v1.1.6.md, line 67 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70Te5xSZDdSF4VcNcB:-L70hdctdY_A2BmuwPwG:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v1.1.6.md#L67)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary><blockquote>

Just this one (prevent sql connection errors while a node is starting up). I misread things earlier. 
</blockquote></details>

Done.

---

*[releases/v1.1.6.md, line 68 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70TXkktHL5s1SQXIJM:-L70hd6JqJyCef1cxw3u:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v1.1.6.md#L68)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary><blockquote>

Don't mention this in the notes. 
</blockquote></details>

Done.

---

*[releases/v2.0-beta.20180312.md, line 39 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70TvNyFzGR7OCvo05d:-L712iTvUITvXdEPVVD9:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v2.0-beta.20180312.md#L39)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary><blockquote>

More capitalization needed in this file.
</blockquote></details>

Done.

---

*[releases/v2.0-beta.20180312.md, line 84 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70USeqRVWdAx5L97rd:-L70hgQorQ7FF1jnDqFt:bwv6yod) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v2.0-beta.20180312.md#L84)):*
<details><summary><i>Previously, bdarnell (Ben Darnell) wrote…</i></summary><blockquote>

I think we're also going to want cockroachdb/cockroach#23541 which is closely related, so the two can share one release note. cc @arjunravinarayan 
</blockquote></details>

OK, @arjunravinarayan, please just post the note here when ready.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2604)*

**Comment by arjunravinarayan** [comment]
*[releases/v2.0-beta.20180312.md, line 84 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70USeqRVWdAx5L97rd:-L7QCJxnmzzogxsDKGfo:baw9n7t) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v2.0-beta.20180312.md#L84)):*
<details><summary><i>Previously, jseldess (Jesse Seldess) wrote…</i></summary><blockquote>

OK, @arjunravinarayan, please just post the note here when ready.
</blockquote></details>

@jseldess #23541 has the release note that can be used for both.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2604)*

**Comment by jseldess** [comment]
*[releases/v2.0-beta.20180312.md, line 84 at r1](https://reviewable.io:443/reviews/cockroachdb/docs/2604#-L70USeqRVWdAx5L97rd:-L7QDmLVLAfleGwvh7A2:bkxcxdg) ([raw file](https://github.com/cockroachdb/docs/blob/64fc05326a6a094a42a9348d8ab6fe26013ae0b8/releases/v2.0-beta.20180312.md#L84)):*
<details><summary><i>Previously, arjunravinarayan (Arjun Narayan) wrote…</i></summary><blockquote>

@jseldess #23541 has the release note that can be used for both.
</blockquote></details>

Done. Thanks, @arjunravinarayan.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/cockroachdb/docs/2604)*

File: releases/v1.1.6.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+### General Changes
+
+- Disabling [diagnostics reporting](../v1.1/diagnostics-reporting.html) also disables new version notification checks. [#23008][#23008]
+
+### Bug Fixes
+
+- Prevented disruptions in performance when gracefully shutting down a node. [#23302][#23302]
+- Fixed a zero QPS scenario when gracefully shutting down a node. [#23306][#23306]
+- Fixed a panic caused by empty `COCKROACH_UPDATE_CHECK_URL` or `COCKROACH_USAGE_REPORT_URL` environment variables. [#23008][#23008]
+- The conversion from [`INTERVAL`](../v1.1/interval.html) to [`FLOAT`](../v1.1/float.html) now
```
Comment (Amruta-Ranade) [inline releases/v1.1.6.md]: nit: Extra newline break

File: releases/v1.1.6.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+  properly returns the number of seconds in the interval. [#22893][#22893]
+- Fixed a bug that caused queries of virtual tables with an explicit database name prefix (e.g. `select * from mydb.crdb_internal.tables`) to not get scrubbed properly in [diagnostics reporting](../v1.1/diagnostics-reporting.html). [#22815][#22815]
+- A `COMMIT` reporting an error generated by a previous parallel statement (i.e., `RETURNING NOTHING`) no longer leaves the connection in an aborted transaction state. Instead, the transaction is considered completed and a `ROLLBACK` is not necessary. [#22814][#22814]
+- Fixed an issue with stale buffer data when using the binary format for [`ARRAY`](../v1.1/array.html) values. [#22262][#22262]
+- Fixed a bug that caused SQL connection errors during node startup. [#22663][#22663]
+- Fixes a bug that included [decommissioned nodes](../v1.1/remove-nodes.html) in cluster stats aggregates. [#22864][#22864]
```
Comment (Amruta-Ranade) [inline releases/v1.1.6.md]: nit: s/Fixes/Fixed

File: releases/v2.0-beta.20180312.md [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+
+### Command-Line Changes
+
+- [Replication zone configs](../v2.0/configure-replication-zones.html) now allow for specifying an ordered list of lease placement preferences. Whenever possible, CockroachDB will attempt to put the lease for a range on a store that satisfies the first set of constraints. If that's not possible, it will attempt to put the lease on a store that satisfies the second set of constraints, and so on. If none of the preferences can be met, the lease will be placed as it has been in the past. [#23372][#23372]
+- When a node spends more than 30 seconds waiting for an `init` command or to join a cluster, a help message now gets prints to `stdout`. [#23181][#23181]
+
+### Admin UI Changes
+
+- Added a new debug page that allows users to create a "custom" graph displaying any combination of metrics. [#23227][#23227]
+- In in geographical map on the homepage of the Admin UI enterprise version, node components now link to **Note Details** page. [#23283][#23283]
```
Comment (Amruta-Ranade) [inline releases/v2.0-beta.20180312.md]: nit: s/In in/In the

File: releases/v2.0-beta.20180312.md [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+
+- Fixed a correctness bug where some `ORDER BY` queries would not return the correct results. [#23541][#23541]
+- The Admin UI no longer hangs after a node's network configuration has changed. [#23348][#23348]
+- The binary format for [`JSONB`](../v2.0/jsonb.html) values is now supported. [#23215][#23215]
+- A node now waits in an unready state for the length of time specified by the `server.shutdown.drain_wait` cluster setting before draining. This helps ensure that load balancers won't send client traffic to a node about to be drained. [#23319][#23319]
+- Fixed a panic when using `UPSERT ... RETURNING` with `UNION`. [#23317][#23317]
+- Prevented disruptions in performance when gracefully shutting down a node. [#23300][#23300]
+- Hardened the [cluster version upgrade](../v2.0/upgrade-cockroach-version.html) mechanism. Rapid upgrades through more than two versions could sometimes fail recoverably. [#23287][#23287]
+- Fixed a deadlock when tables are rapidly created or dropped. [#23288][#23288]
+- Fix a small memory leak in [`IMPORT`](../v2.0/import.html) CSV. [#23259][#23259]
```
Comment (Amruta-Ranade) [inline releases/v2.0-beta.20180312.md]: nit: s/Fix/Fixed

File: releases/v2.0-beta.20180312.md [hunk 5]
```diff
@@ -0,0 +1,135 @@
...
+- Secondary log files (e.g., the SQL execution log) are now periodically flushed to disk, in addition to the flush
+  that occurs naturally when single log files are full (`--log-file-max-size`) and when the process terminates gracefully. Log file rotation is also now properly active for these files. [#23231][#23231]
+- Previously, the `ranges` column in the `node status` command output only included ranges whose raft leader was on the node in question. It now includes the count of all ranges on the node, regardless of where the raft leader is. [#23180][#23180]
+- Fixed a panic caused by empty `COCKROACH_UPDATE_CHECK_URL` or `COCKROACH_USAGE_REPORT_URL` environment variables. [#23008][#23008]
+- Prevented stale reads caused by the system clock moving backwards while the `cockroach` process is not running. [#23122][#23122]
+- Corrected the handling of cases where a replica fails to retrieve the last processed timestamp of a queue. [#23127][#23127]
+- Fixed a bug where the liveness status would not always display correctly on the single-node page in the Admin UI. [#23193][#23193]
+- Fixed bug with showing descriptions on the **Jobs** page in the Admin UI. [#23256][#23256]
```
Comment (Amruta-Ranade) [inline releases/v2.0-beta.20180312.md]: s/Fixed bug with showing/Fixed a bug that showed incorrect descriptions...

File: releases/v1.1.6.md (hunk 1 above)
Comment (jseldess) [reply releases/v1.1.6.md]: Done.

File: releases/v1.1.6.md (hunk 2 above)
Comment (jseldess) [reply releases/v1.1.6.md]: Done

File: releases/v2.0-beta.20180312.md (hunk 3 above)
Comment (jseldess) [reply releases/v2.0-beta.20180312.md]: Done.

File: releases/v2.0-beta.20180312.md (hunk 4 above)
Comment (jseldess) [reply releases/v2.0-beta.20180312.md]: Done.

File: releases/v2.0-beta.20180312.md (hunk 5 above)
Comment (jseldess) [reply releases/v2.0-beta.20180312.md]: Done.

**Pull request merged**